paris/build.gradle

```diff
@@ -22,6 +22,7 @@ android {
 
 dependencies {
     implementation deps.appcompatV7
+    implementation deps.supportV4
```

Comment:
Should be `api` instead of `implementation` since we'll be using `ResourcesCompat` in generated code. Alternatively, and this is probably a better option, we could wrap the call in a function in `ResourcesExtensions` (or create a new `ContextExtensions` since it needs `Context`) and keep this as is.